hypermode.json

```diff
@@ -51,6 +51,11 @@
                     "type": "string",
                     "const": "hypermode",
                     "description": "Host for the model.  Either 'hypermode', or the name of an external host as defined in the 'hosts' section."
+                  },
+                  "mode": {
+                    "type": "string",
```

Comment:
Why not just a boolean `shared: true` (default to false)?

Reply:
(on the go struct, and in the db also)

Reply:
yeah we could do that too, or rather `dedicated: bool` (defaults to false), as we want shared to be the default.

anyways we don't expect to add other options other than "shared" and "dedicated". embedded models are likely going to be added as a different `"host"`, and not as a new `"mode"`, as discussed in the notion doc.

Reply:
just had a chat with Ryan, at some point we would also want to have the distinction between "globally-shared" vs "org-shared".

and so the precedence for when `dedicated: false` (or unset) would be:
- org-shared model
- shared model
- dedicated

so for example, if no org-shared model exists for the specified model, then fallback to shared models, if it also doesn't exist, fallback to dedicated.